components/support/nimbus-fml/test/allow_coenrolling.kts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import android.content.Context as MockContext
+import org.mozilla.experiments.nimbus.MockNimbus
+
+var injected: MockNimbus? = null
+MyNimbus.initialize { injected }
+val feature = MyNimbus.features.homescreen.value()
+
+val coenrollingFeatureIds = MyNimbus.getCoenrollingFeatureIds()
+
+assert(coenrollingFeatureIds == listOf("fun-app-menu"))
+// Record exposure
+MyNimbus.features.nimbusValidation.recordExposure()
+assert(api.isExposed("nimbus-validation"))
```

Comment:
This can be as simple as:

```suggestion
import com.example.app.nimbus.AppNimbus as MyNimbus

val coenrollingFeatureIds = MyNimbus.getCoenrollingFeatureIds()

assert(coenrollingFeatureIds == listOf("fun-app-menu"))

```

Note: that the import of `AppNimbus` needs to match up with the about block in the [`app.fml.yaml` file](https://github.com/mozilla/application-services/pull/5688/files#diff-3f97979c331bb21174f195ea7cdff20170e119929aa00f23bb84075269eb0035).